src/tasks/platforms/minikube.ts

```diff
@@ -79,6 +81,49 @@ export class MinikubeTasks {
           task.title = `${task.title}...${flags.domain}.`
         }
       },
+      {
+        title: 'Checking minikube version',
+        task: async (ctx: any, task: any) => {
+          const version = await this.getMinikbeVersion()
+          const versionComponents = version.split('.')
+          ctx.minikubeVersionMajor = parseInt(versionComponents[0], 10)
+          ctx.minikubeVersionMinor = parseInt(versionComponents[1], 10)
+          ctx.minikubeVersionPatch = parseInt(versionComponents[2], 10)
+
+          task.title = `${task.title}... ${version}`
+        }
+      },
+      {
+        // Starting from Minikube 1.9 there is a bug with storage provisioner which prevents Che from successful deployment.
+        // For more details see https://github.com/kubernetes/minikube/issues/7218
+        // To workaround the bug, it is required to patch storage provisioner as well as its permissions.
+        title: 'Patch minikube storage',
+        enabled: ctx => ctx.minikubeVersionMajor && ctx.minikubeVersionMinor &&
+          ctx.minikubeVersionMajor === 1 && ctx.minikubeVersionMinor >= 9 && ctx.minikubeVersionMinor <= 11,
+        task: async (_ctx: any, task: any) => {
+          // Patch storage provisioner pod to the latest version
+          const storageProvisionerImagePatch = {
+            apiVersion: 'v1',
+            kind: 'Pod',
+            spec: {
+              containers: [
+                { name: 'storage-provisioner', image: 'gcr.io/k8s-minikube/storage-provisioner:latest' }
```

Comment:
Instead of `latest` here we should use use the digest (or a version) to avoid that an update to `latest` breaks things.

Reply:
ok, I will use digest as there is no tag (if don't count `latest`) for the images above `v1.8.1` which is 2 years old.